libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/service/SessionCredentialsService.java

```diff
@@ -147,7 +147,7 @@ public void deleteSessionCredentialsForCri(String ipvSessionId, Cri cri)
                     LogHelper.buildLogMessage(
                             String.format(
                                     "Deleted %d credentials for %s from session credentials table",
-                                    deleted.size(), criId)));
+                                    deleted, criId)));
```

Comment:
It feels a bit weird to return an `int` from `delete` just for this - you could separate `dataStore.getItemsBySortKeyPrefix(ipvSessionId, criId)` into a separate variable and use the size of that instead?